src/editors/radio.js

```diff
@@ -85,6 +85,11 @@ JSONEditor.defaults.editors.radio = JSONEditor.defaults.editors.select.extend({
     this.radioContainer.classList.add('readonly');
     this._super();
   },
+  afterInputReady: function() {
```

Comment:
You should call the parent editor `afterInputReady` instead of calling `self.theme.afterInputReady(self.input);` directly. That way any afterInput code from the parent class is also executed.
````javascript
 afterInputReady: function() {
    var self = this;
    self.input.dataset.containerFor = 'radio';
    self._super();
````

Reply:
Agreed, didn't notice that radio is extending select and not an editor.

Reply:
It works like this: radio class extends select class which extends the editor class.
So radio class inherits all functions from select class which then inherits all functions from editor class.
Calling `this._super()` calls the function from the parent class if you override it in your code.

It is much better to extend the class that matches the functionality, rather than the editor class.
Like the radio editor works with enum, similar to select. So by extending select, all the enum functionality from the prebuild/init functions is already available.

Reply:
Ok, I got rid of `afterInputReady` and we're back to the same codebase 😄